site/en/docs/extensions/mv3/getstarted/tut-quick-reference/index.md

```diff
@@ -0,0 +1,507 @@
+
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/l2HHbaSJeveap8EWLhrU.png", alt="The contents of the extension folder: images folder, manifest.json, service-worker.js, sw-omnibox.js, sw-tips.js,
+and content.js", width="379", height="299" %}
+
+### Load your extension locally {: #locally }
+
+To load an unpacked extension in developer mode, follow the steps in [Development
+Basics][doc-dev-basics-unpacked].
+
+### Open a reference page {: #open-api }
+
+1. Enter the keyword "api" in the browser address bar.
+1. Press "tab" or "space".
+1. Enter the complete name of the API.
+   - OR choose from a list of past searches
+1. A new page will open to the Chrome API reference page.
+
+It should look like this:
+
+<figure>
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/tKsdFmAYFGApMRF47Nlp.gif", alt="Quick API Reference opening the runtime api reference", width="500", height="306", class="screenshot" %}
+  <figcaption>
+  Quick API extension opening the Runtime API.
+  </figcaption>
+</figure>
+
+### Open the tip of the day {: #open-tip }
+
+Click the Tip button located on the navigation bar to open the extension tip.
+
+<figure>
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/GqjdrVtuA0zt87l3QIn9.gif", alt="Open daily tip in ", width="500", height="593", class="screenshot" %}
+  <figcaption>
+  Quick API extension opening the tip of the day.
+  </figcaption>
+</figure>
+
+{% Details %}
+{% DetailsSummary %}
+**Popover API available now in Canary behind a flag**
+{% endDetailsSummary %}
+
+The Popover API is planned for [launch in Chrome 114][popover-chrome-status], available in stable in early May 2023. But you can start using it now by following these steps:
+
+1. Download Canary
+1. Go to chrome://flags
+1. Enable the "Experimental Web Platform features" flag.
+1. Restart the browser
+
+It is enabled by default in Chrome Canary for local testing.
+
+{% endDetails %}
+
+## 🎯 Potential enhancements {: #challenge }
+
+Based on what you’ve learned today, try to accomplish any of the following:
+
+- Explore another way to implement the omnibox suggestions.
+- Create your own custom modal for displaying the extension tip.
+- Open an additional page to the MDN's Web Extensions reference API pages.
+
+## Keep building! {: #continue }
+
+Congratulations on finishing this tutorial 🎉. Continue leveling up your skills by completing other
+tutorials on this series:
+
+| Extension                        | What you will learn                                            |
+|----------------------------------|----------------------------------------------------------------|
+| [Reading time][tut-reading-time] | To insert an element on a specific set of pages automatically. |
+| [Tabs Manager][tut-tabs-manager] | To create a popup that manages browser tabs.                   |
+
+## Continue exploring
+
+To continue your extension service worker learning path, we recommend exploring the following articles:
+
+- TBD
```

Comment:
Have you already got a list of articles to link to here?

Reply:
I was hoping to link to some Service worker "How-to's" or the explainer, but that is only one approach. 

We could also simply link our to general development articles. I'm open to suggestions.